internal/blockchain/client/arbitrum_test.go

```diff
@@ -7,9 +7,9 @@ import (
 
 	api "github.com/coinbase/chainstorage/protos/coinbase/chainstorage"
 
-	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/suite"
 	"go.uber.org/fx"
+	"go.uber.org/mock/gomock"
```

Comment:
The original one is no longer maintained: https://github.com/golang/mock

My first thought was that we need the latest gomock to support latest protobuf; turns out the cause was `proto.Clone` result cannot be compared with reflect.DeepEqual: https://github.com/golang/mock/blob/main/gomock/matchers.go#L116

But since I've already switched to latest version and it works, then why not?

Reply:
`go.uber.org/mock/gomock` is just an import, will be easy to resolve conflicts.

Changes to adding `MatchProto` and fixing function signature for `DoAndReturn` are inevitable.